### ISSUE TITLE
Support dbt v1.8

### ISSUE THREAD

**Issue by davidwitk** [brooklyn-data/dbt_artifacts#426]
## Overview

Current version only allows dbt v1.7: 
```
Required version of dbt for 'dbt_artifacts': ['>=1.3.0', '<1.8.0']
```

When upgrading to dbt v1.8, we get a runtime error. We need to bump the version to support the latest (upcoming) version.

## What problem would this solve?

People can upgrade to dbt v1.8 without being blocked by `dbt_artifacts`.

## Would you be willing to contribute?

- [ ] Yes - I'm on it!
- [x] Yes, but I'd need help getting started.
- [ ] No

**Comment by mwstanleyft** [comment]
Fixed in #428, just needs to be released!

**Comment by jecolvin** [comment]
Just released, dbt Hub should pick it up in ~an hour.
